src/components/element-card/element-card.styles.tsx

```diff
@@ -116,30 +117,35 @@ export const DroppableWrapper = styled.div<IDroppableWrapperProps>`
     width: 100%;
     padding: 0.4rem;
     gap: 0.25rem;
-    border: 1px solid transparent;
+    ${Border.Util.solid({
+        thickness: Border["width-010"],
+        colour: "transparent",
+    })};
     z-index: -1;
 
     ${({ $size }) => generateElementCardView($size)}
 
     ${({ isOver }) =>
         isOver &&
         css`
-            border: 1px dashed ${Color.Primary};
+            ${Border.Util["dashed-default"]({
+                thickness: Border["width-010"],
+                colour: Colour["border-primary"],
```

Comment:
pass radius as param to dashed border

other affected DroppableWrapper in:
- `pill-items-child.styles.tsx`
- `options-child.styles.tsx`